src/service/arguments/action/action-arguments.ts

```diff
@@ -85,7 +85,7 @@ export class ActionArguments {
             startProject: core.getInput("starting-project"),
             loggerLevel: this.getLoggerLevel(core.getInput("logger-level")),
             annotationsPrefix: core.getInput("annotations-prefix"),
-            customCommandTreatment: core.getInput("custom-command-treatment"),
+            customCommandTreatment: core.getInput("custom-command-treatment") === "" ? [] : core.getInput("custom-command-treatment").split(","),
```

Comment:
```suggestion
            customCommandTreatment: core.getInput("custom-command-treatment")?.trim() === "" ? [] : core.getInput("custom-command-treatment").split(","),
```

Reply:
According to `core.getInput` documentation:
Unless trimWhitespace is set to false in InputOptions, the value is also trimmed.